src/app/user/latest-collective/index.tsx

```diff
@@ -0,0 +1,152 @@
+import { CategoryColumn } from '@/app/proposals/components/table-columns/CategoryColumn'
```

Comment:
This file should be placed inside a `sections` folder. Ok we don't have that strcuture yet since the `user` folder is quite old. But please try to follow the intended pattern, rename it to `LatestCollective` for now. Also, index files should only contain `exports` to improve discoverability and reduce verbosity when importing.

Reply:
let's fix that later